### PR TITLE
ST Multisensor Fixes

### PR DESCRIPTION
**Pull request by matchett808-gh**
resolves: 
- https://github.com/Koenkk/zigbee-herdsman-converters/issues/1188 (my issue)
- https://github.com/Koenkk/zigbee2mqtt/issues/3379

I'll need to fix this PR after the Hive one is merged

### CODE REVIEW

File: devices.js [hunk 1]
```diff
@@ -4589,13 +4625,18 @@ const devices = [
...
         meta: {configureKey: 1},
         configure: async (device, coordinatorEndpoint) => {
             const endpoint = device.getEndpoint(1);
             await bind(endpoint, coordinatorEndpoint, ['msTemperatureMeasurement']);
+            const readPowerConfigClusers = [
+                'batteryVoltage',
+            ];
+            await endpoint.read('genPowerCfg', readPowerConfigClusers);
             await configureReporting.temperature(endpoint);
+            await configureReporting.batteryVoltage(endpoint);
```
Comment (Koenkk) [inline devices.js:4639]: Please try changing:
-  `fz.battery_cr2450` to `fz.battery` 
- `await configureReporting.batteryVoltage(endpoint);` to `await configureReporting.batteryPercentageRemaining(endpoint);`
- `const readPowerConfigClusers = [
                'batteryVoltage',
            ];` to `const readPowerConfigClusers = [
                'batteryPercentageRemaining',
            ];`

**Comment by Koenkk** [comment]
Note to self: after this merge https://github.com/Koenkk/zigbee2mqtt/pull/3434

File: devices.js (hunk 1 above)
Comment (matchett808-gh) [reply devices.js:4639]: I'll need to check this later tonight - but I don't think this sensor gives %age remaining - only battery voltage (it's an old sensor from 2015)

Comment (matchett808-gh) [reply devices.js:4639]: Yeah, it only appears to give batteryVoltage:
```
zigbee2mqtt:error 2020-04-28 23:13:03: Failed to configure '0x000d6f000b19539b', attempt 1 (Error: Read 0x000d6f000b19539b/1 genPowerCfg(["batteryPercentageRemaining"], {"timeout":6000,"manufacturerCode":null,"disableDefaultResponse":true}) failed (Error: Status 'UNSUPPORTED_ATTRIBUTE')

```

Also confirmed by popping a new battery in it

Comment (Koenkk) [reply devices.js:4600]: Thanks for trying!

**Comment by Koenkk** [comment]
Thanks!

**Pull request merged**